addons-l10n/en/op-badge.json

```diff
@@ -0,0 +1,4 @@
+{
+  "op-badge/op": "Original poster",
+  "op-badge/project-author": "Project creator"
```

Comment:
```suggestion
  "op-badge/op": "Original Poster",
  "op-badge/project-author": "Project Creator"
```
Title case is consistent with similar labels in Scratch ("Scratch Team", "Forum Moderator").

Reply:
Is this capitalization necessary in project comments?

Reply:
Most strings on the project page also use title case: "Add to Studio", "Copy Link", "Load More".

Reply:
It's not 100% consistent though: "See inside", "View all"

Reply:
> It's not 100% consistent though: "See inside", "View all"

True, but mxmou's examples are more closely related to this.